src/pgen/rhs_tester.cpp

```diff
@@ -20,10 +20,10 @@ void ProblemGenerator(MeshBlock *pmb, ParameterInput *pin) {
   auto &rc = pmb->meshblock_data.Get();
 
   PackIndexMap imap;
-  auto v = rc->PackVariables({fluid_prim::density, fluid_prim::velocity,
-                              fluid_prim::energy, fluid_prim::bfield, fluid_prim::ye,
-                              fluid_prim::pressure, fluid_prim::temperature},
-                             imap);
+  auto v = rc->PackVariables(
```

Comment:
I don't have plans to use it. @jdolence's call

Reply:
I initially wrote the `rhs-tester`, not @jdolence ;) I think it's served it's purpose for now... but it could be a useful test if we ever lose confidence that we're doing things sanely in the fluid sector I suppose.

Reply:
OK lets just leave it in for now then, we can always do a separate hygiene PR later